.github/workflows/dotnet.yml

```diff
@@ -0,0 +1,52 @@
+name: .NET
+on:
+  workflow_dispatch:
+permissions:
+  contents: write
+jobs:
+  build:
+    if: github.repository == 'Rans4ckeR/cncnet-server'
```

Comment:
Is this line needed at all? And that should be changed if we're going to merge it.

Reply:
The structure is restored and the if condition is removed.